src/explore-education-statistics-admin/src/pages/release/content/components/ReleaseContent.tsx

```diff
@@ -345,13 +355,17 @@ const ReleaseContent = () => {
 
       <hr />
 
-      <ReleaseHeadlines release={release} />
+      <ReleaseHeadlines
+        release={release}
+        transformFeaturedTableLinks={transformFeaturedTableLinks}
```

Comment:
I wonder if we'll end up making this more generic in the future if we need to rewrite more than just Featured Table links - at which point would we add more `transformXXX` props, or change this one to be a generic `transformContent` prop which then knows how to target the content that it needs to transform?

Reply:
Just pondering, not asking for any changes btw :)

Reply:
Possibly, we already transform glossary links, but they act the same in all locations so we don't need to pass down a transform function for them from parent components. Something for future us to ponder if we get to that point.